api/v1alpha1/echoapi_types.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/3scale/saas-operator/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	echoapiDefaultImage defaultImageSpec = defaultImageSpec{
+		Name:       pointer.StringPtr("quay.io/3scale/echoapi"),
+		Tag:        pointer.StringPtr("v1.0.1"),
```

Comment:
```suggestion
		Tag:        pointer.StringPtr("v1.0.3"),
```

Bump echo-api to 1.0.3